fortran/test/fetch_content_integration/test_micm_box_model.F90

```diff
@@ -19,14 +20,14 @@ subroutine box_model()
     integer(c_int)     :: solver_type
     integer(c_int)     :: num_grid_cells
 
-    real(c_double), parameter :: GAS_CONSTANT = 8.31446261815324_c_double ! J mol-1 K-1
+    real(real64), parameter :: GAS_CONSTANT = 8.31446261815324_real64 ! J mol-1 K-1
 
-    real(c_double) :: time_step
-    real(c_double), target :: temperature(1)
-    real(c_double), target :: pressure(1)
-    real(c_double), target :: air_density(1)
-    real(c_double), target :: concentrations(6)
-    real(c_double), target :: user_defined_reaction_rates(2)
+    real(real64) :: time_step
+    real(real64), target :: temperature(1)
+    real(real64), target :: pressure(1)
+    real(real64), target :: air_density(1)
+    real(real64), target :: concentrations(6)
+    real(real64), target :: user_defined_reaction_rates(2)
```

Comment:
Should we remove target attributes?

Reply:
I think these have to be `target` now, because the `solve` function expects arrays with the `target` attribute because it uses `c_loc` on them

Reply:
Ah I missed that. Sorry!